### PR TITLE
Issue #158809 chore: Add com_tjvendors as client for country,city,region database

### PR DESCRIPTION
**Pull request by deepa-g**
- Add new column name as `com_tjvendors`  in tj_country, tj_region and tj_city table.
- Update existing dtabase

### CODE REVIEW

File: administrator/sql/country.sql [hunk 1]
```diff
@@ -23,250 +24,250 @@ CREATE TABLE IF NOT EXISTS `#__tj_country` (
...
-(242, 'St. Barthelemy', 'XSB', 'XB', 'TJ_GEO_COUNTRY_SAINTBARTHELEMY', 238, 1, 1, 1),
-(245, 'Aland Islands', 'ALA', 'AX', 'TJ_GEO_COUNTRY_ALANDISLANDS', 239, 1, 1, 1),
-(246, 'Guernsey', 'GGY', 'GG', 'TJ_GEO_COUNTRY_GUERNSEY', 240, 1, 1, 1),
-(247, 'Saint Martin (French part)', 'MAF', 'MF', 'TJ_GEO_COUNTRY_SAINTMARTIN', 241, 1, 1, 1),
-(248, 'Timor-Leste', 'TLS', 'TL', 'TJ_GEO_COUNTRY_TIMORLESTE', 242, 1, 1, 1),
-(249, 'Serbia', 'SRB', 'RS', 'TJ_GEO_COUNTRY_SERBIA', 243, 1, 1, 1),
-(250, 'Isle of Man', 'IMN', 'IM', 'TJ_GEO_COUNTRY_ISLEOFMAN', 244, 1, 1, 1),
-(251, 'Montenegro', 'MNE', 'ME', 'TJ_GEO_COUNTRY_MONTENEGRO', 245, 1, 1, 1),
-(252, 'Palestinian Territory, Occupied', 'PSE', 'PS', 'TJ_GEO_COUNTRY_PALESTINIANTERRITORYOCCUPIED', 246, 1, 1, 1);
+INSERT INTO `movkz_tj_country` (`id`, `country`, `country_3_code`, `country_code`, `country_jtext`, `ordering`, `com_jgive`, `com_jticketing`, `com_quick2cart`, `com_socialads`, `com_tjlms`, `com_tjvendors`) VALUES
```
Comment (ankush-maherwal) [inline administrator/sql/country.sql]: @deepa-g Do not use hard coaded database names in product sql files

File: administrator/sql/country.sql [hunk 2]
```diff
@@ -23,250 +24,250 @@ CREATE TABLE IF NOT EXISTS `#__tj_country` (
...
+(1, 'Afghanistan',  'AFG',  'AF',   'TJ_GEO_COUNTRY_AFGHANISTAN',   1,  1,  1,  1,  1,  1,  1),
+(2, 'Albania',  'ALB',  'AL',   'TJ_GEO_COUNTRY_ALBANIA',   2,  1,  1,  1,  1,  1,  1),
```
Comment (ankush-maherwal) [inline administrator/sql/country.sql]: @deepa-g Why extra spaces before country codes?

File: administrator/sql/updates/mysql/1.4.3.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `#__tj_country` ADD com_tjvendors tinyint(1)  NOT NULL DEFAULT '1' AFTER com_tjlms;
```
Comment (ankush-maherwal) [inline administrator/sql/updates/mysql/1.4.3.sql]: @deepa-g v1.4.4 is already released why the file name is 1.4.3

File: administrator/sql/country.sql (hunk 2 above)
Comment (deepgawade89) [reply administrator/sql/country.sql]: resolved

File: administrator/sql/updates/mysql/1.4.3.sql (hunk 3 above)
Comment (deepgawade89) [reply administrator/sql/updates/mysql/1.4.3.sql]: resolved

**Pull request closed without merging**